lib/pages/card_page_jp.dart

```diff
@@ -5,23 +5,21 @@ import '../utils/constants.dart';
 import '../utils/Styles.dart';
 
 class CardPageJP extends StatelessWidget {
-  const CardPageJP({
-    Key? key,
-    this.photoLink = UserData.photoLink,
-    this.name = UserData.name,
-    this.job = UserData.job,
-    this.mail = UserData.mail,
-    this.address = UserData.address,
-    this.phone = UserData.phone,
-  }) : super(key: key);
+  const CardPageJP(
+      {Key? key,
```

Comment:
I think you can use ctrl + alt + L to fix this indentation

Reply:
Fixed